sync/livepush.ts

```diff
@@ -37,7 +37,7 @@ export async function startLivepush(opts: {
 
 	chokidar
 		.watch('.', {
-			ignored: /((^|[\/\\])\..|node_modules.*|sync\/.*)/,
+			ignored: /((^|[\/\\])\..|node_modules.*|sync\/.*|test\/.*)/,
```

Comment:
Why do we expect a slash after `sync` or `test` but not after `node_modules` ?

Reply:
I specifically didn't want files that had sync or test in the name to be excluded. I guess the same goes for node_modules but it's less likely. I'll just add it to match.